tests/dist/src/test/java/org/hawkular/apm/tests/dist/TraceServiceITest.java

```diff
@@ -210,7 +210,7 @@ public void testStoreAndRetrieveComplexTraceById() {
     public void testStoreAndQueryAll() {
         Trace trace1 = new Trace();
         trace1.setId("1");
-        trace1.setStartTime(System.currentTimeMillis() - 4000); // Within last hour
+        trace1.setStartTime(TimeUnit.MILLISECONDS.toMicros(System.currentTimeMillis()) - 4000); // Within last hour
```

Comment:
Same issue with converting 4000 to microseconds - if change is applied to other files.